docs/dynamic_records.md

```diff
@@ -220,4 +220,22 @@ Sonar check regions (sonar_regions) possible values:
         connect_timeout: 2
         response_timeout: 10
         rapid_recheck: True
-```
\ No newline at end of file
+```
+
+#### [Azure Health Check Options](https://docs.microsoft.com/en-us/azure/traffic-manager/traffic-manager-monitoring#configure-endpoint-monitoring)
+
+| Key | Description | Default |
+|--|--|--|
+| interval_in_seconds          | This value specifies how often an endpoint is checked for its health  from a Traffic Manager probing agent. You can specify two values here:  30 seconds (normal probing) and 10 seconds (fast probing). If no values  are provided, the profile sets to a default value of 30 seconds. Visit  the [Traffic Manager Pricing](https://azure.microsoft.com/pricing/details/traffic-manager) page to learn more about fast probing pricing. | 30      |
+| timeout_in_seconds           | This property specifies the amount of time the Traffic Manager probing  agent should wait before considering a health probe check to an endpoint a failure. If the Probing Interval is set to 30 seconds, then you can  set the Timeout value between 5 and 10 seconds. If no value is  specified, it uses a default value of 10 seconds. If the Probing  Interval is set to 10 seconds, then you can set the Timeout value  between 5 and 9 seconds. If no Timeout value is specified, it uses a  default value of 9 seconds. | 10 or 9 |
+| tolerated_number_of_failures | This value specifies how many failures a Traffic Manager probing agent  tolerates before marking that endpoint as unhealthy. Its value can range between 0 and 9. A value of 0 means a single monitoring failure can  cause that endpoint to be marked as unhealthy. If no value is specified, it uses the default value of 3. | 3       |
```

Comment:
There are double spaces at various places in the description sentences. Can you replace all of them with single space?

Reply:
Fixing these now

Reply:
`'<,'>s/ \s\+/ /g` - regex is fun!